api/swagger/swaggerDoc.yaml

```diff
@@ -543,6 +726,35 @@ definitions:
             type:
               type: string
               example: t
+  PatronsCreatorDataV03:
+    required:
+      - names
+      - username
+      - pin
+      - address
+    properties:
+      name:
+        type: string
+        example: 'TestFirstName, TestLastName'
```

Comment:
Inconsistent quotes here and throughout the next few lines too.

Reply:
Yeah, not sure but it's valid yaml.